Tools/Pipeline/Program.cs

```diff
@@ -45,11 +45,18 @@ static void Main(string [] args)
 			win.Show (); 
 			var model = new PipelineProject();
 			new PipelineController(win, model);  
+			#if LINUX
```

Comment:
Gtk Pipeline is only built on Mac and Linux so there is no reason for there to include Windows, plus with how the file https://github.com/dellis1972/MonoGame/blob/Newaddin/Tools/Pipeline/Program.cs looks, that part of code is completely skipped on WIndows.